core/src/main/kotlin/org/evomaster/core/search/service/Statistics.kt

```diff
@@ -342,6 +355,37 @@ class Statistics : SearchListener {
         return list
     }
 
+    fun getAIData(): List<Pair> {
+        val list = mutableListOf<Pair>()
+
+        // Check if AI classifier is enabled
+        if (!config.isEnabledAIModelForResponseClassification()) {
+            list.add(Pair("ai_model_enabled", "false"))
+            return list
+        }
+
+        list.add(Pair("ai_model_enabled", "true"))
+
+        // Try to get classifier and its metrics
+        val classifier = aiResponseClassifier
+        if (classifier == null) {
+            list.add(Pair("ai_status", "not_initialized"))
+            return list
+        }
+
+        val model = classifier.viewInnerModel()
+        val metrics = model.estimateOverallMetrics()
+
+        // Add derived metrics
+        list.add(Pair("ai_accuracy", "%.4f".format(metrics.accuracy)))
+        list.add(Pair("ai_precision400", "%.4f".format(metrics.precision400)))
+        list.add(Pair("ai_recall400", "%.4f".format(metrics.recall400)))
+        list.add(Pair("ai_f1Score400", "%.4f".format(metrics.f1Score400)))
+        list.add(Pair("ai_mcc400", "%.4f".format(metrics.mcc)))
```

Comment:
these entries should always be filled. if `classifier` is null, I guess 0 could be used